modulemd/v2/include/modulemd-2.0/modulemd-component.h

```diff
@@ -110,4 +112,18 @@ modulemd_component_set_rationale (ModulemdComponent *self,
 const gchar *
 modulemd_component_get_rationale (ModulemdComponent *self);
 
+
+/**
+ * modulemd_component_equals:
+ * @self: This #ModulemdComponent.
+ * @other: The other #ModulemdComponent being compared.
+ *
+ * Returns: TRUE if all of the objects and variables composing the two
+ * ModulemdComponents are equal. False, otherwise. 
+ *
+ * Since: 2.1
```

Comment:
New functions will be added as a new minor release, so this should be:
```suggestion
 * Since: 2.2
```